packages/mui-base/src/AutocompleteUnstyled/useAutocomplete.js

```diff
@@ -1012,7 +1013,10 @@ export default function useAutocomplete(props) {
       onBlur: handleBlur,
       onFocus: handleFocus,
       onChange: handleInputChange,
-      onMouseDown: handleInputMouseDown,
```

Comment:
I decided to use replace `onMouseDown` with `onClick` because this approach requires less change. 

For example, when `multiple` is enabled, we pass `onDelete: handleTagDelete()` to `Chip`s, which are used for multiple values. `Chip`'s `onDelete` API uses a click event, so dealing with event bubbling gets more complicated.